pkg/monitoring/perfscale/vmi-phase-transitions.go

```diff
@@ -164,7 +172,35 @@ func newVMIPhaseTransitionTimeFromCreationHistogramVec(informer cache.SharedInde
 
 	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
 		UpdateFunc: func(oldVMI, newVMI interface{}) {
-			updateVMIPhaseTransitionTimeFromCreationTimeHistogramVec(histogramVec, oldVMI.(*v1.VirtualMachineInstance), newVMI.(*v1.VirtualMachineInstance))
+			vmi := newVMI.(*v1.VirtualMachineInstance)
+			if vmi.DeletionTimestamp == nil {
+				updateVMIPhaseTransitionTimeFromCreationDeletionTimeHistogramVec(histogramVec, oldVMI.(*v1.VirtualMachineInstance), vmi)
+			}
```

Comment:
I don't think we should be considering anything related to the deletion timestamp here for this histogram anchored to creation.

The `from creation` phase transition histogram should work orthogonal to anything related to the deletionTimestamp.